packages/icons/stories/icons.stories.mdx

```diff
@@ -22,19 +23,324 @@ import SvgIcon, { SvgIconTypes } from '../src';
 <Subtitle>FDK SvgIcon component displays inline svg icons.</Subtitle>
 
 <Heading>Installation</Heading>
-<Source code={`npm i ${name}`} language='bash' />
+<Source code={`npm i ${name}`} language="bash" />
 
 <Heading>Usage</Heading>
-<Source code={`import ${SvgIcon.displayName} from '${name}';`} language='ts' />
+<Source code={`import ${SvgIcon.displayName} from '${name}';`} language="ts" />
 
-<Subheading>SvgIcon</Subheading>
-<Subtitle />
+<Subheading>Icons</Subheading>
 
 <Canvas>
-  <Story name='SvgIcon'>
-    <div style={{ padding: '10px' }}>
-      <SvgIcon name='checkStroke' className='checkStroke-icon' />
-    </div>
+  <Story name="applicationWindowStroke">
+    <SC.Container>
+      <SvgIcon
+        name="applicationWindowStroke"
+        className="applicationWindowStroke-icon"
+      />
+    </SC.Container>
+  </Story>
+  <Story name="bookBookmarkStroke">
+    <SC.Container>
+      <SvgIcon name="bookBookmarkStroke" className="bookBookmarkStroke-icon" />
```

Comment:
Bruk SvgIconTypes? Hva brukes className til?

Reply:
Oja, litt usikker, for det er ikke jeg som skrev det. Jeg bare kopierte det som var der fra før.